agent/pyproject.toml

```
@@ -43,6 +42,9 @@ env = [
     "LM2_AGENT_AUTH0_AUDIENCE = str",
     "LM2_AGENT_AUTH0_CLIENT_ID = str",
     "LM2_AGENT_AUTH0_CLIENT_SECRET = str",
+    "LM2_AGENT_BACKEND_BASE_URL = http://backend",
+    "LM2_AGENT_LMSTAT_BIN_PATH = ./tests/mock_tools",
+    "LM2_AGENT_RLMSTAT_BIN_PATH = ./tests/mock_tools",
```

I don't think we have these envs set in the main branch, why is it needed?

```suggestion
    "LM2_AGENT_LMUTIL_PATH = ./tests/mock_tools",
    "LM2_AGENT_RLMUTIL_PATH = ./tests/mock_tools",
```

---

This is pytest configuration. It sets the environment variables for pytest.

---

The env vars for flexlm and rlm binaries were renamed, so I think they should match the new names here.